src/theme/DocSidebar/index.js

```diff
@@ -83,35 +93,41 @@ function DocSidebarItem({item, onItemClick, collapsible}) {
 
 // Calculate the category collapsing state when a page navigation occurs.
 // We want to automatically expand the categories which contains the current page.
-function mutateSidebarCollapsingState(item, location, level) {
+function mutateSidebarCollapsingState(item, path) {
```

Comment:
Please revert this file to the original one as we modified the Sidebar theme to handle the sidebar expansion properly.